Dockerfile-prd

```diff
@@ -1,14 +0,0 @@
-FROM heroku/heroku:18
```

Comment:
Nuking this docker file, as its not needed anymore with the new CI setup and multi layered docker container build with `Dockerfile-build`